devel/logcat-colorize/Portfile

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+name                logcat-colorize
+version             0.8.1
+license             Apache-2
+categories          devel
+platforms           darwin
+maintainers         @carlonluca \
+                    gmail.com:carlon.luca
```

Comment:
You need to use braces `{`and `}`around these to link them. Also, put them on the same line and add `openmaintainer`.